assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -45,14 +45,54 @@ describe( 'core/modules modules', () => {
 		sharedOwnershipModules: [ 'analytics', 'search-console', 'tagmanager' ],
 	};
 
-	const recoverableModuleFixtures = [
+	const moduleFixturesAll = [
```

Comment:
We can probably just drop the "all" suffix here; the `moduleFixturesExternal` suffix makes sense but I think here it's better to leave it out.

```suggestion
	const moduleFixtures = [
```

Or change the variables to reflect how you'd talk about them in English, eg:

```suggestion
	const allModulesFixture = [
```

Reply:
Will go with `allModulesFixture ` cuz `modulesFixture` is already in use